src/utils/db.js

```diff
@@ -0,0 +1,13 @@
+const { Sequelize } = require('sequelize');
+
+require('dotenv').config();
+
+const sequelize = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_NAME,
+  dialect: 'postgres',
+});
```

Comment:
Consider adding a `port` configuration if your database is running on a non-default port. Use `port: process.env.DB_PORT` to specify it.